package.json

```diff
@@ -83,66 +88,14 @@
 			"ie 11"
 		]
 	},
-	"jest": {
-		"roots": [
-			"<rootDir>/src"
-		],
-		"collectCoverage": true,
-		"coverageDirectory": "coverage/jest",
-		"collectCoverageFrom": [
-			"src/**/*.{js,jsx,ts,tsx}",
-			"!src/**/*.test.{js,jsx,ts,tsx}",
-			"!src/**/*.d.ts",
-			"!cypress/**/*.{spec,test}.{js,jsx,ts,tsx}"
-		],
-		"coverageThreshold": {
-			"global": {
-				"branches": 63,
-				"functions": 72,
-				"lines": 78
-			}
-		},
-		"setupFiles": [
-			"react-app-polyfill/jsdom",
-			"<rootDir>/jest-test-setup.js"
-		],
-		"setupFilesAfterEnv": [
-			"<rootDir>/src/setupTests.js"
-		],
-		"testMatch": [
-			"<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
-			"<rootDir>/src/**/*.test.{js,jsx,ts,tsx}"
-		],
-		"testEnvironment": "jest-environment-jsdom-global",
-		"transform": {
-			"^.+\\.(js|jsx|ts|tsx)$": "<rootDir>/node_modules/babel-jest",
-			"^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
-			"^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
-		},
-		"transformIgnorePatterns": [
-			"[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$",
-			"^.+\\.module\\.(css|sass|scss)$"
-		],
-		"modulePaths": [],
-		"moduleNameMapper": {
-			"^react-native$": "react-native-web",
-			"^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
-		},
-		"moduleFileExtensions": [
-			"web.js",
-			"js",
-			"web.ts",
-			"ts",
-			"web.tsx",
-			"tsx",
-			"json",
-			"web.jsx",
-			"jsx",
-			"node"
-		],
-		"watchPlugins": [
-			"jest-watch-typeahead/filename",
-			"jest-watch-typeahead/testname"
+	"babel": {
+		"presets": [
+			[
+				"react-app",
+				{
+					"absoluteRuntime": false
+				}
+			]
 		]
```

Comment:
If there is a babel config file, why is this in package.json. One should be picked over the other, and an external babel.config.js is better than package.json. I am glad to see jest.config.js was finally able to be moved out.